index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Debas Portfolio | Dbus</title>
+</head>
+<body>
+    <main class="container">
+      <header>
+        <nav class="navbar" id="nav">
+            <a href="#" class="logo">Dbus-G'</a>
+            <div class="menu">
+                <a href="#" class="humberger"><i class="fas fa-bars"></i></a>
+            </div>
+        </nav>
+      </header>
+      <section class="headline-section">
```

Comment:
- [ ] Please ensure that your **headline-section** is aligned with **`flexbox`**. In your CSS, that does not seem to be the case.